src/components/playlist-countdown/_playlist-countdown.scss

```diff
@@ -1,5 +1,15 @@
+@keyframes slideIn {
+  0% {
```

Comment:
0% - 50% is for delay? 
if so, use the `animation-delay` property:
`animation: slideIn 500ms 500ms` should work

Reply:
i want it to start at -160px. if i use delay instead then it will be at right 0px while fading in

Reply:
so return `right: -160px` in `.hidden`